mlrun/api/api/endpoints/migrations.py

```diff
@@ -0,0 +1,35 @@
+import http
+
+import fastapi
+
+import mlrun.api.crud
+import mlrun.api.initial_data
+import mlrun.api.schemas
+import mlrun.api.utils.background_tasks
+from mlrun.utils import logger
+
+router = fastapi.APIRouter()
+
+
+@router.post(
+    "/migrations/start",
+    responses={
+        http.HTTPStatus.OK.value: {},
+        http.HTTPStatus.ACCEPTED.value: {"model": mlrun.api.schemas.BackgroundTask},
+    },
+)
```

Comment:
No auth here, which is justified (at least for now, we may need in the future to re-consider who has permissions to do this action) but should at least log this so we remember this is an intentional decision.

Reply:
There is auth, to be more exact authentication, see the `dependencies` definition in `api.py`, in the endpoints themselves we declare it only if we want to actually use the auth verifier instance
yes, we don't have authorization - added a comment about it